contribution_plan/models.py

```diff
@@ -1,3 +1,114 @@
+import uuid
+
+from django.db import models
+from core.models import InteractiveUser
+from django.conf import settings
 from django.db import models
+from core import models as core_models, fields
+from graphql import ResolveInfo
+from jsonfallback.fields import FallbackJSONField
+from location.models import Location
+from insuree.models import Insuree
+from policy.models import Policy
+from product.models import Product
+
+
+class ContributionPlanBundle(core_models.UUIDVersionedModel):
+    id = models.AutoField(db_column='ContributionPlanBundleId', primary_key=True)
+    uuid = models.CharField(db_column='ContributionPlanBundleUUID', max_length=24, default=uuid.uuid4, unique=True)
+    version = models.IntegerField()
+
+    active = models.BooleanField(db_column='Active')
+    code = models.CharField(db_column='ContributionPlanBundleCode', max_length=255, blank=True, null=True)
+    name = models.CharField(db_column='ContributionPlanBundleName', max_length=255, blank=True, null=True)
+
+    json_ext = FallbackJSONField(db_column='Json_ext', blank=True, null=True)
+
+    date_created = fields.DateTimeField(db_column="DateCreated")
+    date_updated = fields.DateTimeField(db_column="DateUpdated")
+
+    user_updated = models.ForeignKey(core_models.InteractiveUser, db_column="UserUpdatedUUID",
+                                     related_name="%(class)s_UpdatedUUID", on_delete=models.deletion.DO_NOTHING)
+    user_created = models.ForeignKey(core_models.InteractiveUser, db_column="UserCreatedUUID",
+                                     related_name="%(class)s_CreatedUUID", on_delete=models.deletion.DO_NOTHING)
+
+    date_valid_from = fields.DateTimeField(db_column="DateValidFrom")
+    date_valid_to = fields.DateTimeField("DateValidTo")
+
+    class Meta:
+        managed = False
```

Comment:
Why not turning on Django management ?

Reply:
It was not necessary, I've restored default management